services/grizzly-jersey/src/main/java/org/swecourse/services/Main.java

```diff
@@ -38,8 +38,8 @@
     port = Optional.ofNullable(System.getenv("SERVICES_GJ_PORT"));
     BASE_URI = PROTOCOL + host.orElse("localhost") + ":" + port.orElse("80") + "/";
     LSTN_URI = PROTOCOL + lstn.orElse("0.0.0.0") + ":" + port.orElse("80");
-    String uname = "steve";
-    String password = "blue";
+    //String uname = "steve";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://178.128.191.195:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)